src/text_render.rs

```diff
@@ -238,8 +207,8 @@ impl TextRenderer {
 
                     let bounds_min_x = text_area.bounds.left.max(0);
                     let bounds_min_y = text_area.bounds.top.max(0);
-                    let bounds_max_x = text_area.bounds.right.min(screen_resolution.width as i32);
-                    let bounds_max_y = text_area.bounds.bottom.min(screen_resolution.height as i32);
+                    let bounds_max_x = text_area.bounds.right.min(resolution.width as i32);
+                    let bounds_max_y = text_area.bounds.bottom.min(resolution.height as i32);
```

Comment:
I wonder if we could get rid of these `min` here? That would remove the `viewport` argument from `prepare` altogether.

Reply:
Users can always set a `scissor_rect` explicitly on their `RenderPass`.

Reply:
The use case I was thinking of here is when people provide many text bounds that are outside the viewport. I'd like to skip generating that geometry entirely instead of using a scissor for that. e.g., panning around a large document that's zoomed in to a small section.

Reply:
Makes sense. I didn't notice `glyphon` was culling glyphs. Scrollables in `iced` are probably benefitting from this!